authoring/authoring.cadl

```diff
@@ -26,6 +26,13 @@ using Azure.Core;
 @route("/authoring/analyze-text/")
 namespace Azure.Language.Authoring;
 
+model AuthFlow {
+  type: OAuth2FlowType.clientCredentials;
+  tokenUrl: "https://textanalytics.authoring.com/oauth2/token";
+  refreshUrl: "https://textanalytics.authoring.com/oauth2/refresh";
```

Comment:
use `https://login.microsoftonline.com/oauth2/...` for the 2. It is not correct but better than this.